recipes/seqcluster/meta.yaml

```diff
@@ -30,6 +30,7 @@ requirements:
     - progressbar2
     - biopython
     - mirtop >=0.4.17a
+    - viennarna
```

Comment:
Is this needed? It is not listed in the `requirements.txt` file.

Reply:
yes, it is. I need to update the requirements as well. I always use conda, so I forget about that one. Thanks for looking at it.